README.md

```diff
@@ -1,11 +1,13 @@
 # Puppet-allknowingdns
 
-[![Build Status](https://travis-ci.org/voxpupuli/puppet-allknowingdns.png?branch=master)](https://travis-ci.org/voxpupuli/puppet-allknowingdns)
-[![Puppet Forge](http://img.shields.io/puppetforge/v/voxpupuli/allknowingdns.svg)](https://forge.puppetlabs.com/voxpupuli/allknowingdns)
+[![Build Status](https://github.com/voxpupuli/puppet-allknowingdns/workflows/CI/badge.svg)](https://github.com/voxpupuli/puppet-allknowingdns/actions?query=workflow%3ACI)
+[![Release](https://github.com/voxpupuli/puppet-allknowingdns/actions/workflows/release.yml/badge.svg)](https://github.com/voxpupuli/puppet-allknowingdns/actions/workflows/release.yml)
+[![Puppet Forge](https://img.shields.io/puppetforge/v/puppet/allknowingdns.svg)](https://forge.puppetlabs.com/puppet/allknowingdns)
 [![Puppet Forge - downloads](https://img.shields.io/puppetforge/dt/puppet/allknowingdns.svg)](https://forge.puppetlabs.com/puppet/allknowingdns)
 [![Puppet Forge - endorsement](https://img.shields.io/puppetforge/e/puppet/allknowingdns.svg)](https://forge.puppetlabs.com/puppet/allknowingdns)
 [![Puppet Forge - scores](https://img.shields.io/puppetforge/f/puppet/allknowingdns.svg)](https://forge.puppetlabs.com/puppet/allknowingdns)
-[![License](http://img.shields.io/:license-gpl3-blue.svg)](http://www.gnu.org/licenses/gpl-3.0.html)
+[![puppetmodule.info docs](http://www.puppetmodule.info/images/badge.png)](http://www.puppetmodule.info/m/puppet-allknowingdns)
```

Comment:
What is this puppetmodule.info?  Does not resolve to a valid address here… :thinking: 

```sh-session
romain@zappy ~ % host www.puppetmodule.info               
www.puppetmodule.info is an alias for www.puppetmodule.info.herokudns.com.
```

Reply:
Cannot run a worldwide test…

![RIPE Atlas screenshot saying I can't create a measurement to check this worldwide](https://user-images.githubusercontent.com/148721/208530864-699c1af8-f5a6-42ce-8035-d058cf34d44f.png)

Reply:
it's currently having troubles and will hopefully come back tomorrow, from a new owner/server! (spoiler: https://github.com/voxpupuli/puppetmodule.info)

Reply:
I **knew** I saw something matching that name recently…